.idea/.gitignore

```diff
@@ -0,0 +1,8 @@
+# Default ignored files
+/shelf/
+/workspace.xml
+# Datasource local storage ignored files
+/../../../../../../:\Users\Connor\IdeaProjects\finalProject\.idea/dataSources/
+/dataSources.local.xml
+# Editor-based HTTP Client requests
+/httpRequests/
```

Comment:
This needs help.  The .gitignore for this project should be verbose.  I suspect that's why one commit changed 131 files... the gitignore should filter these out.